packages/kit-bg/src/migrations/indexedToBucketsMigration/legacyIndexedDb.ts

```
@@ -19,31 +19,35 @@ import type { IIndexedDBSchemaMap } from '../../dbs/local/types';
+                transaction,
+              });
+            },
           },
-        },
-      );
-      resolve(legacyDb);
-    });
-    await this.legacyDb;
+        );
+        resolve(legacyDb);
+      });
```

Biome (1.9.4)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)